src/scheduler/data_types.h

```diff
@@ -339,6 +345,7 @@ struct server_info
 	 */
 	status *policy;
 	fairshare_head *fairshare;	/* root of fairshare tree */
+	resresv_set **equiv_classes;
```

Comment:
I disagree with the name change.  The run only one example could use the resresv_set structure, the jobs are not equivalent.  The equiv_sets are a set of jobs that are equivalent to each other.  In run only one, they are specifically going to be different.